server/app.js

```diff
@@ -199,8 +199,9 @@ app.use(function(err, req, res, next) {
 // Add cronjob to automatically refresh aliases
 var dbtype = nconf.get('database:type')
 if (dbtype == 'mysql') {
+  var cronartime = nconf.get('database:aliasRefreshInterval');
```

Comment:
Sorry one other change - if the config item is not present (like in upgraded instances) this will fail, needs logic to fall back on a default